### DISCUSSION TITLE
Filtering by click on labels

### DISCUSSION THREAD

**Discussion by marvinwankersteen** [astubenbord/paperless-mobile#186] [Feature Requests]
**Describe the bug**
It's no longer possible to filter documents by click/tab on the correspondent in the main view.
Instead it's possible by click/tab on the tags.

If I choose manual a correspondent over the filterbutton, and press then on the correspondent, the filter is cleared. But it's not cleared if filtered by a tag.

**To Reproduce**
1. Go to 'Documents'
2. Click on the choosen correspondent. E.g. "Mr. X"
3. Nothing happens

**Expected behavior**
Documents should be filtered by the correspondent "Mr. X".

**Device Information:**
 - Device: OnePlus 5, Samsung Galaxy Tab S5e
 - OS: LOS20 with microG, LOS20 crDroid
 - Paperless Mobile version: 2.3.2+2037
 - Paperless Server version: 1.14.5

In the previous version 2.1.0 it worked.

PS: I love this app 👍

**Comment by astubenbord** [comment]
Yeah, sorry I took that feature out because the hitboxes of both correspondent and document type are too small to intuitively consider them as an element of interaction. Is that something you use on a regular basis?


PS: Thanks for the compliment :)

**Comment by marvinwankersteen** [comment]
Ah, okay.
I had already noticed that you have to press very precisely. However, I got used to it and had no more problems.

I actually used it a lot, because I find the manual search for the correspondent more cumbersome until you have selected or found the right one in the list.

As an alternative, would it be possible to set the filter when you have opened a document? The tags and the correspondent are also displayed there.

Or maybe an option in the settings to enable/disable this feature?

**Comment by astubenbord** [comment]
The mechanisms/callbacks to achieve this behavior are still present in the current codebase and it wouldn't require much effort to implement in both ways (click directly on the label in on the list tile and on the opened document).I'll convert this to a feature request.

**Comment by schtibal** [comment]
I second putting this back in!  I actually love this feature about the mobile app! I actually submitted a feature against Paperless-ngx to add something similar (before I actually realised it worked on your mobile app).

It's a very quick and convenient way to drill down to the right document through tags. If I start with one tag, I'll be presented with a heap of documents that match that tag. Then as I go through the docs presented, I can keep tapping tags to add them to the filter criteria until I can find the doc that I want.

**Reply by astubenbord** [reply]
Will be in the next update today :)
